enhancements/metering/slim-install.md

```diff
@@ -0,0 +1,210 @@
+
+This approach would allow metering to interact with more raw metric data, in addition to a more streamlined data flow. The process of integrating this connector will require a full overhaul of the existing promQL and SQL queries/views defined throughout the default set of `ReportDataSource` and `ReportQuery` resources, in addition to rethinking how storage would need to be handled.
+
+In theory, metering can utilize existing CRD functionality to periodically store daily partitions of Prometheus data in Presto schemas. One avenue that could be explored is the usage of scheduled `Reports`, which metering stores streamed Prometheus data in daily partitions in Presto. Alternatively, a `INSERT INTO ... SELECT * FROM ... WHERE timestamp > .status.lastTimestamp` type operation could be entertained and achieve similar results.
+
+Once that initial integration has happened, it's possible to remove (i.e. stop reconciling) the Hive components (metastore and server). Presto, an in-memory query engine, still requires the usage of a metastore, but it's possible to have Presto use a local, file-based metastore and back up that path in the container with a PersistentVolumeClaim (PVC). With this path, the reporting-operator would always create tables in Presto instead of Hive, i.e. always favor creating `PrestoTable` resources, and the functionality provided by the Hive components would be obsolete.
+
+A couple of things to call out with this approach:
+
+- The Hive components, and especially Hive server are fairly idle in the current architecture due to the nature of periodically importing metric data.
+- Removing the Hive components could reduce the installation complexity as the presence of hard-to-come-by storage access modes, like RWM, will no longer be needed.
+- As aforementioned, Presto requires the usage of metastore, and favoring a local, file-based one instead of a more robust and dedicated metastore implementation could be problematic.
+- The current implementation of AWS billing integration requires the usage of Hive server, namely to manage report partitions. Note: maintaining support for AWS billing is listed as out-of-scope for this proposal.
+- Setting reasonable resource limits and configuration the JVM properly for Presto can be challenging and Presto is wired to use all resources that are available to it.
+- Need to determine a support matrix between previous versions of the operator with this new integration.
+
+Overall, this alternative stands out the most for providing a more streamlined in-cluster metering solution, in addition, to providing ways to slim down that implementation.
+
+### Integrate with the Prometheus remote_write API
+
+Utilize the Prometheus remote_write/remote_read configuration and one of the ecosystem's exporters. This would allow metering to swap out the Hive/Presto stack, and be able to handle more raw, accurate metric data than what the Prometheus query_range API can offer.
```

Comment:
Same thing, I'm pretty sure accuracy is not at issue, just the ability to ask for and get what metrics the query wants when it queries, as opposed to the export always model.